src/common/utils/time-utils.test.ts

```diff
@@ -115,3 +117,50 @@ describe('subtract time', () => {
     );
   });
 });
+
+describe('get start of time', () => {
+  test('Start of hour', () => {
+    expect(getStartOf(new Date(2020, 2, 1, 0, 30), TimeUnit.HOURS)).toEqual(
+      testDate,
+    );
+  });
+  test('Start of day', () => {
+    expect(getStartOf(new Date(2020, 2, 1, 8, 30), TimeUnit.DAYS)).toEqual(
+      testDate,
+    );
+  });
+  // Start of week test will need to be adjusted when implementing Luxon migration as
+  // moment treats Sunday as start of week and luxon treats Monday as start of week.
+  // Currently, no start of week function calls are implemented.
+  test('Start of week', () => {
+    expect(getStartOf(new Date(2020, 2, 5), TimeUnit.WEEKS)).toEqual(testDate);
+  });
+  test('Start of month', () => {
+    expect(getStartOf(new Date(2020, 2, 15), TimeUnit.MONTHS)).toEqual(
+      testDate,
+    );
+  });
+});
+
+describe('get time difference', () => {
+  test('Difference in hours', () => {
+    expect(
+      getTimeDiff(testDate, new Date(2020, 2, 1, 8), TimeUnit.HOURS),
+    ).toEqual(-8);
+  });
+  test('Difference in days', () => {
+    expect(getTimeDiff(testDate, new Date(2020, 2, 5), TimeUnit.DAYS)).toEqual(
+      -4,
+    );
+  });
+  test('Difference in weeks', () => {
+    expect(
+      getTimeDiff(testDate, new Date(2020, 2, 15), TimeUnit.WEEKS),
+    ).toEqual(-2);
+  });
+  test('Difference in months', () => {
+    expect(
+      getTimeDiff(testDate, new Date(2020, 5, 1), TimeUnit.MONTHS),
+    ).toEqual(-3);
```

Comment:
Consider adding a test case for positive diffs? Just one or two examples where the first date passed to `getTimeDiff` is after the date passed as second argument